docs/pipelines.md

```diff
@@ -606,6 +608,106 @@ tasks:
       name: build-push
 ```
 
+### Using the `onError` field
+
+> :seedling: **Specifying `onError` in `PipelineTasks` is an [alpha](additional-configs.md#alpha-features) feature.** The `enable-api-fields` feature flag must be set to `"alpha"` to specify `onError`  in a `PipelineTask`.
+
+> :seedling: This feature is in **Preview Only** mode and not yet supported/implemented.
+
+When a `PipelineTask` fails, the rest of the `PipelineTasks` are skipped and the `PipelineRun` is declared a failure. If you would like to
+ignore such `PipelineTask` failure and continue executing the rest of the `PipelineTasks`, you can specify `onError` for such a `PipelineTask`.
```

Comment:
What about the case when the Task produces a result that may be needed downstream or by the pipeline?

Reply:
Wouldn't setting it to just "continue" without providing a value to the outputs of the task make it fail?

Reply:
Sorry! I spoke too soon. You already go over that use case below.